lib/chef/knife/bootstrap_windows_base.rb

```diff
@@ -114,6 +114,12 @@ def self.included(includer)
             :long        => "--[no-]node-verify-api-cert",
             :description => "Verify the SSL cert for HTTPS requests to the Chef server API.",
             :boolean     => true
+
+          option :msi_url,
+            :short => "-u URL",
+            :long => "--msi_url URL",
+            :description => "Location of the Chef Client MSI. The default templates will prefer to download from this location. The MSI will be downloaded from chef.io if not provided.",
```

Comment:
We don't fallback, to another location, so there isn't a download preference to prefer, right?
```
Location of the Chef Client MSI to be installed by the bootstrap template. Default is to download from chef.io.
```